src/prerender.ts

```diff
@@ -164,13 +167,15 @@ export async function prerender(nitro: Nitro) {
 
     // Fetch the route
     const encodedRoute = encodeURI(route);
-    const res = await (localFetch(
+
+    const res = await localFetch<Response>(
       withBase(encodedRoute, nitro.options.baseURL),
       {
         headers: { "x-nitro-prerender": encodedRoute },
+        retry: nitro.options.prerender.retries ?? 2, // default is 2 retries
+        retryDelay: nitro.options.prerender.retryDelay ?? 250, // default is 250ms
```

Comment:
Looks like the fallback retry amount and delay don't match the default values stated in the docs

Reply:
done